tests/db/test_migration_check.py

```diff
@@ -24,12 +25,13 @@ def test_migration_check():
     Make sure there is a database dump for the latest version of the db and
     that a migration test exists for this dump.
     """
-    versions_folder: Path = Path(".").absolute() / "src" / "inmanta" / "db" / "versions"
+    inmanta_dir: str = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
Why use `os.path` here and `Path` below?

Reply:
I didn't realize what I needed from `os.path` was present in `Path` as well.
I uniformized it, using `Path` everywhere.